app/src/main/java/com/github/godspeed010/weblib/LibraryFragment.kt

```diff
@@ -151,18 +178,12 @@ class LibraryFragment : Fragment() {
 
     private fun focusEditText(et: EditText) {
         et.requestFocus()
-        val imm = context?.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager?
-        imm?.toggleSoftInput(InputMethodManager.SHOW_FORCED, InputMethodManager.HIDE_IMPLICIT_ONLY)
+        hideKeyboard()
```

Comment:
Oops, I guess I just glanced over it and since it looks very similar to the original hide keyboard code, I had just replaced it with the hide keyboard function.